src/pudl/transform/ferc1.py

```diff
@@ -694,6 +694,106 @@ def combine_axis_columns_xbrl(
     return df
 
 
+class CheckTableCalculations(TransformParams):
+    """Parameters for checking xbrl-metadata based calculations within a table."""
+
+    column_to_check: str | None = None
+    """Name of data column to check.
+
+    This will typically be ``dollar_amount`` or ``ending_balance`` in hte
+    """
+    calculation_tolerance: float = 0.05
+    """The tolerance ratio of the off calcuations and the possible calcuations."""
```

Comment:
It's common for these kinds of tolerance checks to include both "absolute" (difference) and "relative" (ratio) tolerances (e.g. see the `atol` and `rtol` parameters to `np.isclose()`.  Even if we aren't using both types of tolerance here I think it would be helpful if the parameter name indicated which type we're talking about (relative in this case)

Reply:
i hear you but this is not the tolerance of the calced-value vs the reported-value, but rather the # of calculations that are `~np.isclose` w/ default `atol` and `rtol` vs the total number of calculated values.

although another name for this is encouraged!